examples/to_do.rs

```diff
@@ -1,6 +1,7 @@
 use gtk::prelude::*;
-use relm4::factory::{DynamicIndex, FactoryComponent, FactoryComponentSender, FactoryVecDeque};
-use relm4::{gtk, ComponentParts, ComponentSender, RelmApp, SimpleComponent, WidgetPlus};
+use relm4::factory::FactoryVecDeque;
+use relm4::prelude::*;
+use relm4::RelmApp;
```

Comment:
I think we should probably add RelmApp to the prelude too. What do you think?

Reply:
Sure, that one seems relatively low-risk since the type name is already "namespaced".

Reply:
Added.